apps/i18n/bounce/az_az.json

```diff
@@ -56,7 +56,7 @@
   "pathRight": "əgər sağa yol varsa,",
   "pilePresent": "təpəcik var",
   "playSoundBounce": "sıçrayış səsi çıxart",
-  "playSoundCheer": "play cheering sound",
+  "playSoundCheer": "",
```

Comment:
How come some files we'll show a blank string? But others we'll remove the key too? Or is the Greek file an exception/bug?

Reply:
This is just the way crowdin does it; it looks like yml files remove the key and json files just drop it down to an empty string.

I'm eventually going to look into adding a step to our out sync to remove these, but they're harmless.